cmd/server/main.go

```diff
@@ -49,7 +49,9 @@ func (api *deploymentAPI) registerHandler(w http.ResponseWriter, r *http.Request
 	platform := r.FormValue("platform")
 	installerType := r.FormValue("installerType")
 	apiToken := r.FormValue("apiToken")
-
+	if apiToken == "" {
+		apiToken = r.Header.Get("Authorization")
```

Comment:
This will return the whole "Api-Token 123abc" part. You'll need to split that spring with something like:
```go
strings.Split(r.Header.Get("Authorization"), " ")[1]
```

Reply:
Now it is fixed.